index.js

```diff
@@ -39,8 +39,13 @@ function addNumbers(num1, num2) {
  *   email: "leia@leia.com",
  * }
 */
-function makePersonObject(/* code here */) {
+function makePersonObject(id, name, email) {
   /* code here */
+  return {
+    "id" : id,
+    "name": name,
+    "email": email
+  };
```

Comment:
For objects, if the value to be assigned is the same as the object property it is being assigned to, you can simply just do something like this: 
```js
{
  id,
  name,
  email,
}
```

check out this [link](https://alligator.io/js/object-property-shorthand-es6/) for more context